app/models/checkout.rb

```diff
@@ -51,6 +51,24 @@ def self.longest_checkout_time_today(offset)
     Time.at(difference).utc.strftime('%H:%M:%S')
   end
 
+  def self.longest_checkout_game_today(offset)
+    start_time = (Time.now - 15.hours).strftime('%Y-%m-%d %H:%M:%S')
+    min_checkout = self.where(closed: false).where(
+                       "(check_out_time - '#{offset} hours'::interval) > ?",
+                       start_time
+                     ).order(:check_out_time).first
+    if min_checkout
```

Comment:
This should be more concise:

```ruby
  def self.longest_checkout_game_today(offset)

    # ...rest of logic above min_checkout

    if min_checkout
      min_game = Game.find(min_checkout.game_id)
      min_game_checkout = {
        min_checkout: min_checkout,
        min_game: min_game.name
      }
    end
  end
```

You don't need the else / final return value.